erpnext/accounts/doctype/purchase_invoice_item/purchase_invoice_item.json

```diff
@@ -591,7 +591,6 @@
    "oldfieldname": "purchase_order",
    "oldfieldtype": "Link",
    "options": "Purchase Order",
-   "print_hide": 1,
```

Comment:
purchase_order should be hidden from print.

Reply:
It's useful to have a reference for the other party to get a proper trail.